<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateFields</class>
 <widget class="QDialog" name="UpdateFields">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Terminal Finances::Atualizar Lançamento</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>531</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Editar o lançamento de código: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_id">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>99</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_value_edit_balance">
          <property name="text">
           <string>Valor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="value_edit_balance">
          <property name="maximum">
           <double>999999.989999999990687</double>
          </property>
          <property name="singleStep">
           <double>1.000000000000000</double>
          </property>
          <property name="stepType">
           <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_desc_edit_balance">
        <property name="text">
         <string>Descrição</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fiel_desc_edit_balance"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="styleSheet">
         <string notr="true">color:#fff;
background-color:#dc3545;
border-color:#dc3545;
font-weight:bold;</string>
        </property>
        <property name="text">
         <string>Excluir</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_edit_balance">
        <property name="styleSheet">
         <string notr="true">color:#fff;
/*background-color:#28a745;*/
background-color:#57B4C0;
border-color:#28a745;
font-weight:bold;</string>
        </property>
        <property name="text">
         <string>Editar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
